pkg/apis/objectbucket.io/v1alpha1/objectbucketclaim_types.go

```diff
@@ -40,6 +39,9 @@ type ObjectBucketClaimSpec struct {
 	// AdditionalConfig gives providers a location to set
 	// proprietary config values (tenant, namespace, etc)
 	AdditionalConfig map[string]string `json:"additionalConfig"`
+	// ObjectBucketName is the name of the object bucket resource.  This is the authoritative
+	// determintaion for binding.
+	ObjectBucketName string
```

Comment:
Looking at the [design doc](https://github.com/kube-object-storage/lib-bucket-provisioner/blob/master/doc/design/object-bucket-lib.md#obc-custom-resource-after-update-by-lib) this field was supposed to exist in the OBC but be named `objectBucketRef`.

Reply:
True... i think that's probably due to the docs becoming stale as the code changed.  In the end I believe we decided a ObjectReference inner struct was overkill.